src/esp/sensor/Sensor.h

```diff
@@ -19,22 +19,22 @@ namespace sensor {
 
 // Enumeration of types of sensors
 enum class SensorType {
-  NONE = 0,
-  COLOR = 1,
-  DEPTH = 2,
-  NORMAL = 3,
-  SEMANTIC = 4,
-  PATH = 5,
-  GOAL = 6,
-  FORCE = 7,
-  TENSOR = 8,
-  TEXT = 9,
+  None = 0,
+  Color = 1,
+  Depth = 2,
+  Normal = 3,
+  Semantic = 4,
```

Comment:
I believe a couple of places use the `Semantic` and the `Depth` as well. But I did not see the changes in the files below.